src/Common/src/Interop/FreeBSD/Interop.Libraries.cs

```diff
@@ -5,6 +5,12 @@ internal static partial class Interop
 {
     private static partial class Libraries
     {
-        internal const string LibRt = "librt"; // POSIX Realtime Extensions library
+        /// <summary>
+        /// We aren't OS X so don't have an INODE64 suffix to entry points
+        /// </summary>
+        internal const string INODE64SUFFIX = "";
+
+        internal const string LibRt = "librt";  // POSIX Realtime Extensions library
+        internal const string libc = "libc";    // C runtime
```

Comment:
Why is libc being defined in each of these Interop.Libraries.cs files... isn't it already in the Unix/Libraries.Interop.cs file?  Are we not including that for some reason?